static/app/views/settings/account/notifications/notificationSettings.spec.tsx

```diff
@@ -74,4 +74,22 @@ describe('NotificationSettings', function () {
     }
     expect(screen.getByText('Issue Alerts')).toBeInTheDocument();
   });
+
+  it('renders spend section instead of quota section with feature flag', async function () {
+    const {routerContext, organization} = initializeOrg({
+      organization: {
+        features: ['slack-overage-notifications', 'spend-visibility-notifications'],
+      },
+    });
+
+    renderMockRequests({});
+
+    render(<NotificationSettings organizations={[organization]} />, {
```

Comment:
Is it useful to have a test ensuring this behavior when you have a mix of orgs with and without spend-visibility feature?